proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +47,23 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  // owner_name is the name of the owner of the repository,
+  // either a username or organization name.
+  string owner_name = 10;
+  // description is the user configurable description of the repository.
+  string description = 11;
+  // url is the user configurable URL in the description of the repository,
+  // always included the scheme and will not have a #fragment suffix.
+  string url = 12;
+  // license is the user configurable license of the repository, which should be one of
+  // the identifier defined in https://spdx.org/licenses
+  string license = 13;
+  // license url is the user configurable url to the license text.
+  string license_url = 14;
```

Comment:
Is license not going to be a license file someone pushes, but a url they have to specify? What if the license of a repo/proto changes over time?

Reply:
do you mean we should read the LICENSE file on `buf push` instead of making it selectable by user?

Reply:
If we want to have docs on pkg.go.dev, the generated packages zips need to have a license file in them. At least in the goproxy, license is a per commit thing and not a repository wide setting. If a user would change their license for new commits, I'm doubtful that we should generate zips with this newer license for commits that were pushed earlier.